File: ocaml/fizzbuzz.ml
let fizzbuzz n =
  if n mod 15 = 0 then "FizzBuzz"
  else if n mod 3 = 0 then "Fizz"
  else if n mod 5 = 0 then "Buzz"
  else string_of_int n

let () =
  for i = 1 to 100 do
    fizzbuzz i |> print_endline
  done

File: python/fizzbuzz.py
for n in range(1, 101):
    s = ''
    is_div_3 = n % 3 == 0
    is_div_5 = n % 5 == 0
    if is_div_3 or is_div_5:
        if is_div_3:
            s += 'Fizz'
        if is_div_5:
            s += 'Buzz'
    else:
        s = n
    print(s)
